cvat/apps/engine/annotation.py

```diff
@@ -117,7 +143,35 @@ def save_task(tid, data):
             }
 
     for jid, _data in splitted_data.items():
-        save_job(jid, _data, True)
+        # if an item inside _data isn't empty need to call save_job
+        isNonEmpty = False
+        for action in ['create', 'update', 'delete']:
+            for objects in _data[action].values():
+                if objects:
+                    isNonEmpty = True
+                    break
+
+        if isNonEmpty:
+            save_job(jid, _data, True)
```

Comment:
What will be happened if data from single job are separated to two chunks?
Third "True" argument might remove data from previous request.

Reply:
Why do we need erase old data if we called "delete" request from client before?
isNonEmpty flag doesn't appear pretty